test/Engine/ProtoTest/EventTests/EventTest.cs

```diff
@@ -1,220 +1,256 @@
-using System;

```

Comment:
GitHub's diff algorithm is really really dumb at times.

Could you stick a note on the diff as to what has actualy changed here?